js-extensions/spec/ExtensionRenderer-spec.js

```diff
@@ -90,21 +90,25 @@ describe('ExtensionRenderer', function () {
 
     it('should change the wrapping element', function () {
         const result = mount(React.createElement(ExtensionRenderer, {extensionPoint: 'ep1', wrappingElement: 'section'}));
+        console.log(result.html());
```

Comment:
why do you add that debug code? That should be configurable, since it has the potential to trash the test output big time.